.github/workflows/gradle.yml

```diff
@@ -40,6 +39,10 @@ jobs:
     - name: Update SHA
       working-directory: ./tmp/pa
       run: ./gradlew updateShas
+    - name: Set docker-compose path
+      run: echo ::set-env name=DOCKER_COMPOSE_LOCATION::$(which docker-compose)
+    - name: Set vm.max_map_count
+      run: sudo sysctl -w vm.max_map_count=262144
     - name: Start Build
       working-directory: ./tmp/pa
-      run: ./gradlew build
+      run: ./gradlew build --info --stacktrace
```

Comment:
The info creates a lot of details and the actual errors are somewhere up. Do we really want this ?

Reply:
Removed